### DISCUSSION TITLE
VM

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1203] [Q&A]
Hi, I know the Mac M1 cannot install the VM client successfully. So I thought if I install the Ubuntu in VM machine on the mac m1 laptop. Is it can work VM client ? thanks

**Comment by ataffanel** [comment]
Hi,
The Bitcraze VM is an AMD64 VM and so it cannot run on Mac M1 at the moment. Unfortnuatly the client is also curently broken on the latest mac-os (https://github.com/bitcraze/crazyflie-clients-python/issues/694) and we have not had time to look at it yet (if you want to give it a go, we will gladly merge a pull request!).

So, if you have Mac OS Sonoma, you cannot run the client at the moment. Earlier version of Mac OS would work on a mac M1

**Comment by knmcguire** [comment]
A fix has been proposed for the Mac M1 issue that can solve the vispy issue temporary https://github.com/bitcraze/crazyflie-clients-python/issues/694
